tests/suite_utils.py

```
@@ -0,0 +1,169 @@
+
+
+def zero_downtime_worker(stop_evt, error_evt, instance, ssl=False, key="test"):
+    logging.info("Starting zero-downtime worker")
+    try:
+        db = instance.create_connection(ssl=ssl, force_reconnect=True)
+        g = db.select_graph(key)
+        while not stop_evt.is_set():
+            g.query("CREATE (n:Person {name: 'Alice'})")
+            g.ro_query("MATCH (n:Person {name: 'Alice'}) RETURN n")
+            time.sleep(2)
+    except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)